x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/tours/translation_guide/index.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import type { PopoverAnchorPosition } from '@elastic/eui';
+import { EuiButtonEmpty, EuiTourStep } from '@elastic/eui';
+import { noop } from 'lodash';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { DocLink } from '../../../../../common/components/links_to_docs/doc_link';
+import { useIsElementMounted } from '../../../../../detection_engine/rule_management_ui/components/rules_table/rules_table/guided_onboarding/use_is_element_mounted';
+import {
+  NEW_FEATURES_TOUR_STORAGE_KEYS,
+  SecurityPageName,
+} from '../../../../../../common/constants';
+import { useKibana } from '../../../../../common/lib/kibana';
+import { SIEM_MIGRATIONS_STATUS_HEADER_ID } from '../../rules_table_columns';
+import { SIEM_MIGRATIONS_SELECT_MIGRATION_BUTTON_ID } from '../../header_buttons';
+import * as i18n from './translations';
+
+export const SECURITY_GET_STARTED_BUTTON_ANCHOR = `solutionSideNavCustomIconItem-${SecurityPageName.landing}`;
+
+const tourConfig = {
+  currentTourStep: 1,
+  isTourActive: true,
+  tourPopoverWidth: 360,
+};
+
+const tourSteps: Array<{
+  step: number;
+  title: string;
+  content: React.ReactNode;
+  anchorPosition: PopoverAnchorPosition;
+}> = [
+  {
+    step: 1,
+    title: i18n.MIGRATION_RULES_SELECTOR_TOUR_STEP_TITLE,
+    content: i18n.MIGRATION_RULES_SELECTOR_TOUR_STEP_CONTENT,
+    anchorPosition: 'downCenter',
+  },
```

Comment:
tiny nit: How come you don't store the anchors in here too
```suggestion
  {
    step: 1,
    title: i18n.MIGRATION_RULES_SELECTOR_TOUR_STEP_TITLE,
    content: i18n.MIGRATION_RULES_SELECTOR_TOUR_STEP_CONTENT,
    anchorPosition: 'downCenter',
    anchor: `#${SIEM_MIGRATIONS_SELECT_MIGRATION_BUTTON_ID}`
  },
```

Reply:
Good point! Addressed in https://github.com/elastic/kibana/pull/207425/commits/37d58466514d18490f8a976061dd2aba098d3a8e